tests/acceptance/SubscriberManagementCest.php

```diff
@@ -74,7 +75,8 @@ function deleteGlobalSubscriber(\AcceptanceTester $I) {
     $new_subscriber_email = 'deleteglobaluser99@fakemail.fake';
     $this->generateSingleSubscriber('deleteglobaluser99@fakemail.fake', 'Delete', 'ThisGlobalUser');
     $I->login();
-    $I->amOnMailPoetPage ('Subscribers');
+    $I->amOnMailPoetPage('Subscribers');
+    $I->waitForElementNotVisible('.mailpoet_listing_loading');
```

Comment:
What about creating a helper function in `tests/_suppoert/AcceptanceTester.php` for this waiting. Something like `$I->waitForListingItemsToLoad()`? I think that this is what it does and from that name it would be more obvious and also if we decide to change implementation (e.g. wait for some different element) we will be able to change it in one place.